packages/datasource/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need a top-level package.json if in any case we include the datasources explicitely?

Reply:
yes unfortunately